backend/app/app/models/semester.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime, timezone
+
+from .base import Base
+
+
+class Semester(Base):
+    id: int
+    name: str
+    start: datetime
+    end: datetime
+    choice_deadline: datetime
+
+    @property
+    def is_enroll_open(self):
+        return datetime.now(timezone.utc) < self.choice_deadline
```

Comment:
Please check if in the DB a UTC time is stored instead of UTC+3. If so, leave a comment on it in the code, otherwise, consider the following:

either use `timezone(timedelta(hours=3))` or (preferably):
Config.py:
```
timezone = pytz.timezone(os.getenv("TIMEZONE"))  # set TIMEZONE=Europe/Moscow 
```

Usage:
```
datetime.now(tz=config.timezone)
```